### ISSUE TITLE
Omron G3MB 202P SSR - library is WRONG

### ISSUE THREAD

**Issue by sledzik1984** [ManjuprasadB/eagle_library#1]
![omron](https://user-images.githubusercontent.com/1233199/38465311-a9be1eda-3b1a-11e8-925b-7a198641f56e.jpg)

As you can see - this library is wrong.

**Comment by erazor83** [comment]
ACK.
I've printed my PCB to make sure it fits.
Positions of the switch are: 7.62 / 10.16 and not 5.08 / 7.62.
Look here: https://media.digikey.com/pdf/Data%20Sheets/Omron%20PDFs/G3MB.pdf

**Comment by basaravia** [comment]
yes, i have the same Problem, solved using, "relay.lbr" - item AQG12105 

https://www.diymodules.org/eagle-show-library?type=std&file=relay.lbr
